tests/sqlparser_snowflake.rs

```diff
@@ -4082,3 +4084,16 @@ fn parse_connect_by_root_operator() {
         "sql parser error: Expected an expression, found: FROM"
     );
 }
+
+#[test]
+fn test_snowflake_create_view_with_tag() {
+    let create_view_with_tag = r#"CREATE VIEW X (COL WITH TAG (pii='email')) AS SELECT * FROM Y"#;
+    snowflake().verified_stmt(create_view_with_tag);
+}
+
+#[test]
+fn test_snowflake_create_view_with_tag_and_comment() {
+    let create_view_with_tag_and_comment =
+        r#"CREATE VIEW X (COL WITH TAG (pii='email') COMMENT 'foobar') AS SELECT * FROM Y"#;
+    snowflake().verified_stmt(create_view_with_tag_and_comment);
+}
```

Comment:
Can we merge both scenarios into a single test? e.g called `test_snowflake_create_view_options`